tests/foreman/destructive/test_installer.py

```diff
@@ -17,13 +17,29 @@
 :Upstream: No
 """
 import pytest
+from fauxfactory import gen_domain
+from fauxfactory import gen_string
 
 from robottelo.config import settings
 from robottelo.utils.installer import InstallerCommand
 
 pytestmark = pytest.mark.destructive
 
 
+@pytest.fixture
+def set_wrong_fqdn(target_sat, request):
```

Comment:
Technically speaking this fixtures sets a shortname as a hostname (as Debian does by default btw). So I don't think this sets it as "wrong", but rather something like `with_random_short_hostname`.

Reply:
updated the name.